dependencies.gradle

```diff
@@ -116,6 +117,7 @@ ext.libraries = [
         recyclerAnimators    : "jp.wasabeef:recyclerview-animators:$versions.recyclerAnimators",
         materialProgress     : "me.zhanghai.android.materialprogressbar:library:$versions.materialProgress",
         circleImageView      : "de.hdodenhof:circleimageview:$versions.circleImageView",
+        bottomNavigationView : "com.github.ittianyu:BottomNavigationViewEx:$versions.bottomNavigationView",
```

Comment:
and it has commented code [here](https://github.com/ittianyu/BottomNavigationViewEx/blob/master/widget/src/main/java/com/ittianyu/bottomnavigationviewex/BottomNavigationViewEx.java#L56-L69) and [here](https://github.com/ittianyu/BottomNavigationViewEx/blob/master/widget/src/main/java/com/ittianyu/bottomnavigationviewex/BottomNavigationViewEx.java#L455)

Reply:
I think it is dangerous to add this library to our project, because it can broke something on any device. So, the best way is to implement it yourself.

We can add it if it has really good reason to exist.

Reply:
Ok, I'll look for another implementation.